kaggle_environments/__init__.py

```diff
@@ -40,4 +40,7 @@
             "specification": getattr(env, "specification"),
         })
     except Exception as e:
-        print("Loading environment %s failed: %s" % (name, e))
+        if "football" not in name:
+            print("Loading environment %s failed: %s" % (name, e))
+        else:
+            print("Loading environment %s failed: %s. This is safe to ignore unless trying to use gfootball." % (name, e))
```

Comment:
Do we need to print anything at all, or can we just fully ignore it instead?

Reply:
99.9% ignore? I don't think anyone is building this environment anymore

Reply:
will remove